rust/private/rustc.bzl

```diff
@@ -635,7 +635,7 @@ def establish_cc_info(ctx, crate_info, toolchain, cc_toolchain, feature_configur
         # bazel hard-codes a check for endswith((".a", ".pic.a",
         # ".lib")) in create_library_to_link, so we work around that
         # by creating a symlink to the .rlib with a .a extension.
-        dot_a = ctx.actions.declare_file(crate_info.name + ".a", sibling = crate_info.output)
+        dot_a = ctx.actions.declare_file("lib" + crate_info.name + ".a", sibling = crate_info.output)
```

Comment:
Since you have this code in your brain, is there a reason why we don't use the same logic here and in the toolchain.bzl to create symlinks? Could we do the same?

Reply:
That sounds good! I'll extract a helper function and use it from both places.